scripts/core/helpers/data-provider.ts

```diff
@@ -21,6 +27,7 @@ export class DataProvider implements IDataProvider {
     listeners: Array<() => void> = [];
 
     private scheduleUpdate: () => void;
+    private cache: IRestApiResponse<any>;
```

Comment:
It's be better to add [some generic types](https://user-images.githubusercontent.com/2076953/129571676-66865acb-4ad2-494e-b1a7-9626513e7de9.png) instead of `any`.